src/cmd/manage_nsfs.js

```diff
@@ -223,38 +223,26 @@ async function add_bucket(data) {
 /** verify_bucket_owner will check if the bucket_owner has an account
  * after it finds one, it returns the account id, otherwise it would throw an error
  * (in case the action is add bucket it also checks that the owner has allow_bucket_creation)
- * @param {string} bucket_owner account name
+ * @param {string} bucket_owner - bucket owner is the account name
  * @param {string} action
  */
 async function verify_bucket_owner(bucket_owner, action) {
-    let is_bucket_owner_exist = false;
-    let is_allow_bucket_creation = false;
-    let account_id;
-    const show_secrets = false;
-    const fs_context = native_fs_utils.get_process_fs_context();
-    const entries = await nb_native().fs.readdir(fs_context, accounts_dir_path);
-    // Gap - should replace this implementation
-    // it keeps iterating even if we find that the bucket owner exist
-    await P.map_with_concurrency(10, entries, async entry => {
-        if (entry.name.endsWith('.json') && !is_bucket_owner_exist) {
-            const full_path = path.join(accounts_dir_path, entry.name);
-            const data = await get_config_data(full_path, show_secrets);
-            if (data.name === bucket_owner) {
-                is_bucket_owner_exist = true;
-                is_allow_bucket_creation = data.allow_bucket_creation;
-                account_id = data._id;
-            }
-        }
-    });
-
+    // check if bucket owner exists
+    const fs_context = native_fs_utils.get_process_fs_context(config_root_backend);
```

Comment:
@romayalon I was not sure in the `get_process_fs_context` if I should pass the `config_root_backend`.
I know that you planned to refactor this function and the argument will be `backend`.
In manage_nsfs there are calls to `get_process_fs_context` with `config_root_backend` and without any argument.